resources/user-guides/getting-started-with-adobe-experience-platform/README.md

```diff
@@ -39,16 +39,16 @@ To start collecting data in Adobe Experience Platform, you need to create an XDM
    * Select `Next`
    * Set a name for this dataset and select `Finish`.
 
-### Generate an Experience Edge environment identifier <a id="generate-an-experience-edge-environment-identifier"></a>
+### Generate a Datastream identifier <a id="generate-a-datastream-identifier"></a>
 
-The SDK requires a configuration identifier that ensures the implementation matches the server-side Edge configuration and data is routed/received to/from the correct destination.
+The SDK requires a configuration identifier that ensures the implementation matches the server-side Datastream configuration and data is routed/received to/from the correct destination.
 
 To create a configuration identifier use the following steps:
 
-1. In [Adobe Experience Platform Launch](https://experience.adobe.com/launch), navigate to your mobile property and select **Edge Configurations** from the left panel, then select **New Edge Configuration**.
-2. Provide a name and description and then proceed to set up the default environment settings. These settings are used as defaults across the Experience Edge environments.
+1. In [Adobe Experience Platform Launch](https://experience.adobe.com/launch), navigate to your mobile property and select **Datastreams**  from the left panel, then select **New Datastream**.
```

Comment:
nit: Looks like you added an extra space between Datastreams and from,  "Datastreams  from the left panel, ..."